.travis.yml

```
@@ -2,7 +2,11 @@ sudo: required
 language: python
 python:
   - "3.6.4"
-# command to run tests
+services: 
+  - docker
+before_install: 
+  - docker build -t easycrm -f Dockerfile .
 install:
   - pip install -r requirements.txt
-script: nosetests -v
+script:     
+  - docker run -p 5000:8090 -e CI=true easycrm nosetests -v -- --coverage --watchAll=false 
```

```suggestion
  - docker run -e CI=true easycrm nosetests -v -- --coverage --watchAll=false 
```
My bad, seems like we would not need this port here.